eng/common/TestResources/New-TestResources.ps1

```diff
@@ -18,32 +18,13 @@ param (
     [Parameter(Mandatory = $true)]
     [string] $ServiceDirectory,
 
-    [Parameter(Mandatory = $true)]
+    [Parameter()]
     [ValidatePattern('^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$')]
     [string] $TestApplicationId,
 
     [Parameter()]
     [string] $TestApplicationSecret,
 
-    [Parameter()]
```

Comment:
Was able to eliminate some parameters which the pipelines use but would be unlikely for users to use regularly given the easier alternatives.